buildscripts/kokoro/windows.bat

```diff
@@ -40,4 +41,32 @@ for /r %%F in (TEST-*.xml) do (
 )
 @echo on
 
-exit /b %GRADLEEXIT%
+IF NOT %GRADLEEXIT% == 0 (
+  exit /b %GRADLEEXIT%
+)
+
+@rem make sure no daemons have any files open
+cmd.exe /C "%WORKSPACE%\gradlew.bat --stop"
+
+cmd.exe /C "%WORKSPACE%\gradlew.bat  %GRADLE_FLAGS% -Dorg.gradle.parallel=false -PrepositoryDir=%WORKSPACE%\artifacts\x86_32\ clean grpc-compiler:build grpc-compiler:uploadArchives" || exit /b 1
+
+mkdir grpc-java-helper64
+cd grpc-java-helper64
+call "%VS120COMNTOOLS%\..\..\VC\bin\amd64\vcvars64.bat" || exit /b 1
```

Comment:
It scares me to run these again in the same environment. For example, this script will break if you do 64 bit first, because CMAKE_VSARCH would still be set from the first call. Let's at least change to using CMD to call make_dependencies.bat instead of CALL. To do better we'd have to split this into two extra files I think.